aQute.libg/test/aQute/libg/command/CommandTest.java

```diff
@@ -2,24 +2,45 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+import java.io.File;
+import java.nio.file.Path;
+import java.util.Optional;
+
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 public class CommandTest {
+	private String java = null;
+
+	@BeforeEach
+	void init() {
+
+		java = Optional.ofNullable(
+			aQute.lib.io.IO.JAVA_HOME)
+			.map(h -> h.toPath()
+				.resolve("bin")
+				.resolve("java"))
+			.map(Path::toFile)
+			.filter(File::exists)
+			.filter(File::isFile)
+			.map(File::toString)
+			.orElse("java");
+	}
 
 	@Test
 	public void testSimple() throws Exception {
 		Command c = new Command();
-		c.add("java");
+		c.add(java);
```

Comment:
I just added `IO.getJavaExecutablePath(String name)` to master. So you can rebase and then just do

```java
c.add(IO.getJavaExecutablePath("java"));
```

This also means you wont need the BeforeEach!

Reply:
done